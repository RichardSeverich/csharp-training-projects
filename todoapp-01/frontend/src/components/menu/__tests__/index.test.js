import React from 'react';
import MenuPages from '../.';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

let Menu = () => {
	return (
		<Router>
			<MenuPages />
		</Router>
	);
};

describe('components/menu', () => {
	it('should be instance of', () => {
		const { container } = render(<Menu />);
		expect(container.firstChild).toBeInstanceOf(HTMLDivElement);
	});

	it('should match the snapshot', () => {
		const { container } = render(<Menu />);
		expect(container.firstChild).toMatchSnapshot();
	});
});
