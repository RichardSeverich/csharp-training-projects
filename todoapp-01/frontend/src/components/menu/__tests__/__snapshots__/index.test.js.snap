// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/menu should match the snapshot 1`] = `
<div
  class="ui pointing menu"
>
  <a
    class="item"
    href="/"
  >
    Home
  </a>
  <a
    class="item"
    href="/projects"
  >
    Projects
  </a>
  <a
    class="item"
    href="/view-tasks"
  >
    Tasks
  </a>
  <a
    class="item"
    href="/tags"
  >
    Tags
  </a>
</div>
`;
